starknet_py/net/account/account_client.py

```diff
@@ -332,6 +340,8 @@ async def sign_transaction(
         :param version: Transaction version
         :return: InvokeFunction created from the calls
         """
+        version = version or self.supported_tx_version
```

Comment:
if version is 0 then self.supported_tx_version is used, is this what we want?

Reply:
Definitely not